### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by bill7pearl**
# In this PR I have:
- Added any missing JSX elements to my project (without adding extra functionality).
- Matched Zeplin's design.
- Used the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/bookRenderer.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const BookRenderer = () => {
 
   return (
     <>
-      <div>
+      <div className="books-container">
```
Comment (Felix45) [inline src/components/bookRenderer.js:16]: - [ Required ] Great job, adding the UI styling to the project, however, I think you assumed the drawing canvas was also part of the project design and therefore you introduced extra `paddings` and `margins` on your design. Please see the screenshot below detailing the extra white space `( Arrows in red )` I'm talking about and rectify this.

- [ Required ] The margin between any two books on the list is also wider than on the `Zeplin` design. Kindly rectify this.

## Zeplin design
Screenshot from 2023-02-11 05-16-52

## Your design
Screenshot from 2023-02-11 05-18-10

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @bill7pearl , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :heavy_check_mark: 
- [ ] No linter errors :white_check_mark: 
- [ ] Repository has a professional readme file :white_check_mark: 
- [ ] Gitflow is used correctly :white_check_mark: 
- [ ] PR has a professional title and summary :white_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/bookRenderer.js (hunk 1 above)
Comment (bill7pearl) [reply src/components/bookRenderer.js:16]: done

**Review by franclobo — APPROVED** [review]
Hi @bill7pearl, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Styles are good looking ✔️ 
- All changes requested have been completed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**